src/ISoulbound.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+/// @title An idea for an 1155-flavored SBT / cryptocredential standard
+/// @author neodaoist
+/// @notice Just exploring for now, Keeping it simple and clear
+/// @dev Further ideas — 3rd-Party Attestations, Hubs and Authorities, Reveal/Conceal Pattern
+interface ISoulbound {
+    //
+
+    /*//////////////////////////////////////////////////////////////
+                        Events
+    //////////////////////////////////////////////////////////////*/
+
+    /// @notice Explain to an end user what this does
+    /// @dev Explain to a developer any extra details
+    /// @param _issuer Documents a parameter just like in doxygen (must be followed by parameter name)
+    /// @param _issuee Documents a parameter just like in doxygen (must be followed by parameter name)
+    /// @param _tokenId Documents a parameter just like in doxygen (must be followed by parameter name)
+    event Issue(
+        address indexed _issuer,
+        address indexed _issuee,
+        uint256 _tokenId
+    );
+
+    /// @notice Explain to an end user what this does
+    /// @dev Explain to a developer any extra details
+    /// @param _revoker Documents a parameter just like in doxygen (must be followed by parameter name)
+    /// @param _revokee Documents a parameter just like in doxygen (must be followed by parameter name)
+    /// @param _tokenId Documents a parameter just like in doxygen (must be followed by parameter name)
+    /// @param _reason Documents a parameter just like in doxygen (must be followed by parameter name)
```

Comment:
Boilerplate doc (there's a lot of it in this file, I wont keep duplicating comments)